### DISCUSSION TITLE
What is the equivalent of `nn.map_variables` in the `nnx` API?

### DISCUSSION THREAD

**Discussion by carlesoctav** [google/flax#4464] [General] [answered]
What is the equivalent of `nn.map_variables` in the `nnx` API?

**Comment by cgarciae** [comment, marked as answer]
@carlesoctav you mimic `map_variables` using `nnx.split` to get the pytree state and `nnx.update` to update the Module after you've modified the state. See https://flax.readthedocs.io/en/latest/nnx_basics.html#the-flax-functional-api.
